tests/beaker_tests/cisco_bgp_af/test_bgpaf.rb

```diff
@@ -185,6 +185,35 @@
   resource:      { 'ensure' => 'present' },
 }
 
+tests[:title_patterns_4] = {
```

Comment:
IMO these new title_pattern tests are not needed. All they really do is test the title regex logic and that's already being done by T.1 - T.3.

What I would do for beaker is just add a basic test a la:

```
tests[:l2vpn_evpn] = {
   desc:           '3.1 Address-family l2evpn/evpn',
   platform:         'n(5|6|7|8|9)k',
   title_pattern:  '2 default l2vpn evpn',
   manifest_props: {
     ensure: 'present',
  }
}

test_harness_run(tests, :l2vpn_evpn) unless nexus_i2_image

```